theme/src/main/assets/assets/javascripts/groups.js

```diff
@@ -148,6 +148,41 @@ $(function() {
       });
     });
 
+     // Index Page toc
+     $(".toc").each(function() {
+      var tocRoot = $(this);
+      function findHeaderElements(element) {
+        var headerElements = element.find(".header");
+        headerElements.each(function() {
```

Comment:
with a filter? and then foreach?

Reply:
Yes, that is simpler, I will try it later

Reply:
This fix the `blank` entry, it should be fixed in `css` but fixed in `js`. @mdedetrich @pjfanning @samueleresca
and later should update https://github.com/apache/incubator-pekko-site